Can't install with React 17.0.1

I can't install package with react 17.0.1

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.0 | ^16.14.0" from react-awesome-reveal@3.7.0
npm ERR! node_modules/react-awesome-reveal
npm ERR!   react-awesome-reveal@"^3.7.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "analyze": "size-limit --why"
   },
   "peerDependencies": {
-    "react": "^17.0.0 | ^16.14.0"
+    "react": ">=16.14.0"
   },
   "husky": {
     "hooks": {

